dotnet3.5/S3/ListObjectsExample/ListObjects/ListObjects.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.ListObjects]
```

Comment:
Add a blank line after the license declaration, to preserve the declaration's integrity.

Reply:
Misunderstanding on my part. I put a blank line after the snippet comment. Added.